src/libmeasurement_kit/net/connect.cpp

```diff
@@ -5,14 +5,17 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h" // For MK_CA_BUNDLE
 #endif
+#ifndef MK_CA_BUNDLE
+#error "MK_CA_BUNDLE is not set."
+#endif
```

Comment:
I prefer the build to _fail_ if this is not set. If it does not fail, it silently proceeds and we _may_ end up with a library that does not work correctly because of this reason. This will require to update the Android build, BTW.